src/Runner.Listener/MessageListener.cs

```diff
@@ -268,6 +268,7 @@ public async Task<TaskAgentMessage> GetNextMessageAsync(CancellationToken token)
                 }
 
                 Trace.Info($"Message '{message.MessageId}' received from session '{_session.SessionId}'.");
+                Trace.Info($"Message '{message.MessageId}' char length: '{message.Body.Length}', byte length '{System.Text.Encoding.UTF8.GetByteCount(message.Body)}'.");
```

Comment:
nit: does this add any value now that ProcessChannel.SendAsync traces?

Reply:
I don't think it does.